src/Security/InteractiveSecurityHelper.php

```diff
@@ -12,14 +12,16 @@
 namespace Symfony\Bundle\MakerBundle\Security;
 
 use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Bundle\MakerBundle\Validator;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\User\UserInterface;
 
 /**
  * @internal
  */
 final class InteractiveSecurityHelper
 {
-    public function guessFirewallName(SymfonyStyle $io, array $securityData): string
+    public static function guessFirewallName(SymfonyStyle $io, array $securityData): string
```

Comment:
I'd prefer non-statics. Even if it means we're instantiating it... and then just using it.